### PR TITLE
Konstantinos /  Ubuntu font 

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: static/css/ubuntu.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@font-face {
+    font-family: 'Ubuntu';
+    font-style: normal;
+    font-display: swap;
+	src:url('../fonts/Ubuntu-Regular.ttf') format('truetype');
+	font-weight: 100 700;
```
Comment (habib-deriv) [inline static/css/ubuntu.css:6]: Do we need 100 ?

Comment (konstantinosG-deriv) [reply static/css/ubuntu.css:6]: I guess not that's the value other fonts have in the codebase too. Could go 400 which is default for normal?

File: static/css/ubuntu.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@font-face {
+    font-family: 'Ubuntu';
+    font-style: normal;
+    font-display: swap;
+	src:url('../fonts/Ubuntu-Regular.ttf') format('truetype');
+	font-weight: 400;
```
Comment (habib-deriv) [inline static/css/ubuntu.css:6]:
```suggestion
	font-weight: 700;
```

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**